### PR TITLE
[cherry pick 6.3] Include gfx12 and gfx1151 in default architecutre list for rmake.py

### PR DESCRIPTION
**Pull request by NguyenNhuDi**

### CODE REVIEW

**Review by umfranzw — APPROVED** [review]
This looks good to me. The CI failures don't look like they are related to the change.

**Review by umfranzw — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Full documentation for rocPRIM is available at [https://rocm.docs.amd.com/projec
 ## rocPRIM 3.3.0 for ROCm 6.3.0
 
 ### Added
-
+* Updated the default value for the `-a` argument from `rmake.py` to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```
Comment (spolifroni-amd) [inline CHANGELOG.md]: From the code it looks like the default value has changed to `default_gpus` from `gfx906;gfx1030;gfx1100;gfx1101;gfx1102`

Can you let me know if this is right? Or clarify what has changed?

**Review by spolifroni-amd — CHANGES_REQUESTED** [review]
I have one issue that needs to be addressed.

File: CHANGELOG.md (hunk 1 above)
Comment (NguyenNhuDi) [reply CHANGELOG.md]: Yes, but default_gpus is just a variable holding the string `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`

Comment (spolifroni-amd) [reply CHANGELOG.md]:
```suggestion
* * Changed the default value of `rmake.py -a` to `default_gpus`. This is equivalent to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```

**Review by spolifroni-amd — APPROVED** [review]

**Pull request merged**